myning/tui/chapter/__init__.py

```diff
@@ -166,21 +167,20 @@ def clear(self):
         self.pick(PickArgs(message="", options=[]))
 
 
-def get_labels_and_hotkeys(options: list[OptionLabel]):
+def get_labels_and_hotkeys(options: list[Option]) -> tuple[list[list[str | Text]], dict[str, int]]:
     hotkeys: dict[str, int] = {}
     labels: list[list[str | Text]] = []
-    options = [option if isinstance(option, list) else [option] for option in options]
     for option_index, option_arr in enumerate(options):
-        if not isinstance(option_arr, list):
-            option_arr = [option_arr]
+        if not isinstance(option_arr.label, list):
+            option_arr.label = [option_arr.label]
 
-        if option_index == len(options) - 1:  # No hotkey for last option
-            labels.append(option_arr)
+        if not option_arr.enable_hotkeys:  # No hotkey for last option
```

Comment:
let's keep the logic for checking if its the last thing in the array, so we can have two conditions. sometimes there is only one thing in the list so it also happens to be the last thing, and players can press enter to select it. this will help us not have to worry about whether we should enable hotkeys on an option, reducing cognitive load

Reply:
I think that leaving it would slightly decrease the cognitive load for us, but greatly increase the cognitive load for future contributors who may stumble on a bug where they enable hotkeys and nothing happens. Maybe we could have a `default_option(label: OptionLabel, handler: Handler) -> Option` which would be the same as the normal constructor, but default to enable_hotkeys = False. This could be used on all "Go Back", "Cool cool cool", and "No" options.

Reply:
I don't like the idea of adding another function to return the same thing with just one thing different, but I get your point of it potentially being confusing if enabling hotkeys doesn't work for the last option. It is an established pattern in the game to have the last option be "Go Back"-related stuff though, so how about just adding a docstring to the `enable_hotkeys` property describing this behavior? I think that would be sufficient. 

Or, we could change the current behavior to let it auto-generate hotkeys for the last option; I see no harm in that even though we already have two hotkeys for selecting the last option: `escape` and `q`

Reply:
I made it where the last option is set to enable_hotkeys=False